tests/emailpassword/test_emailverify.py

```diff
@@ -986,13 +989,14 @@ async def test_the_generate_token_api_with_valid_input_verify_and_then_unverify_
     user_id = dict_response["user"]["id"]
 
     verify_token = await create_email_verification_token(user_id)
-    if verify_token.token is None:
-        raise Exception("Should never come here")
-    await verify_email_using_token(verify_token.token)
+    if isinstance(verify_token, CreateEmailVerificationTokenOkResult):
+        if verify_token.token is None:
+            raise Exception("Should never come here")
+        await verify_email_using_token(verify_token.token)
```

Comment:
You have changed the semantics of the test. Also redundant checking happening here.